internal/runtime/init/init_test.go

```diff
@@ -56,6 +56,19 @@ p = true { 1 = 2 }`
 			expectedMods: []string{mod1},
 			asBundle:     false,
 		},
+		{
+			note: "load duplicate files",
+			fs: map[string]string{
+				"datafile":   `{"foo": "bar", "x": {"y": {"z": [1]}}}`,
+				"policyFile": mod1,
+			},
+			loadParams: []string{"datafile", "policyFile", "."},
+			expectedData: map[string]string{
+				"/foo": "bar",
+			},
+			expectedMods: []string{mod1},
+			asBundle:     false,
+		},
```

Comment:
@anderseknert, this test fails, please can you give me pointers on why? It seems the LoadPaths function is not reading all the files in the directory, otherwise, `expectedMods` and `actualMods` should be the same